tests/datasets/test_data_loaders.py

```diff
@@ -0,0 +1,77 @@
+'''
+Test the people class
+'''
+
+#%% Imports and settings
+import sciris as sc
+import covasim.datasets.data_loader as cdt
+import numpy as np
+import json
+from unittest.mock import MagicMock
+import os
+
+dirname = os.path.dirname(__file__)
+
+neherlabs_pop_json_path = os.path.join(dirname, "mock_datasets/population/neher_labs.json")
+neherlabs_pop_translated_json_path = os.path.join(dirname, "mock_datasets/population/neher_labs_translated.json")
+test_json_path = os.path.join(dirname, "mock_datasets/population/test.json")
+with open(neherlabs_pop_json_path) as data:
+    neherlabs_pop_json = json.load(data)
+with open(neherlabs_pop_translated_json_path) as data:
+    neherlabs_pop_translated_json = json.load(data)
+
+neher_labs_raw_pop = open(neherlabs_pop_json_path, "r").read()
+
+def test_transform_neherlab_data():
```

Comment:
Will this test fail if the data changes? Do we want it to?

Reply:
I mocked out all the network calls so the tests don't need to hit an external url. We could have it hit the external url and do the whole workflow if we want to ensure the external script did not change, but I am not sure if we can trust all our test runners to have internet access..

Reply:
i think this is something we'd rerun rarely, if at all, so probably wouldn't include it in the CI test suite (which should have internet access though)

Reply:
I have ci pulling in the population, otherwise the examples fail.